packages/commercetools/api-client/src/fragments/index.ts

```diff
@@ -50,6 +49,61 @@ export const CustomerFragment = `
     firstName
     lastName
     email
+    addresses {
+      id
+    }
+    shippingAddresses {
+      id,
```

Comment:
so create a fragment for address and use it in both places

Reply:
btw. why are you adding something to the query while you don't use it anywhere?

Reply:
OK keep this as it is, but please create an issue for that and link it in the code (comment)